src/AM.Condo/Targets/Prepare/Go.targets

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+  <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="GetGoInfo" Returns="@(GoProjectPath)">
+      <ItemGroup>
+        <GoProjectPath Include="$(RepositoryRoot)" Condition=" Exists('$(RepositoryRoot)go.mod') " />
+        <GoProjectPath Include="$(RepositoryRoot)" Condition=" Exists('$(RepositoryRoot)go.condo') " />
+        <GoTests Include=" Exists('$(RepositoryRoot)*_test.go') " />
+        <GoTests Include=" Exists('$(RepositoryRoot)**$(slash)*_test.go') " />
+      </ItemGroup>
+
+      <PropertyGroup>
+        <GoEnabled Condition=" '$(SKIP_GO)' != '' ">false</GoEnabled>
+        <GoEnabled Condition=" '$(GoEnabled)' != '' AND '$(GoEnabled.ToLower())' != 'true' ">false</GoEnabled>
```

Comment:
this line will handle non-valid values in line 13 as well as anyone who sets GoEnabled in condo.build to a non-valid value -- this is how we've done this elsewhere :)